bytecomp/bytegen.ml

```diff
@@ -576,7 +576,7 @@ let rec comp_expr env exp sz cont =
         in
         comp_init env sz decl_size
       end
-  | Lprim((Pidentity | Popaque), [arg]) ->
+  | Lprim((Pidentity | Popaque | Pbytes_to_string | Pbytes_of_string), [arg]) ->
```

Comment:
I can see there are used.
It just feels inconsistent to only partially add primitives for `bytes`.